components/suggestion/at_mention_provider/at_mention_suggestion.jsx

```diff
@@ -124,6 +126,17 @@ export default class AtMentionSuggestion extends Suggestion {
                     url={Utils.imageURLForUser(item.id, item.last_picture_update)}
                 />
             );
+
+            customStatus = (
+                <CustomStatusEmoji
+                    showTooltip={true}
+                    userID={item.id}
+                    emojiSize={15}
+                    emojiStyle={{
+                        marginLeft: 4,
+                    }}
```

Comment:
since most of the time this is set to `marginLeft: 4` I would suggest adding this setting to the `defaultProps` in `components/custom_status/custom_status_emoji.tsx`